backend/routers/index_router.py

```diff
@@ -16,3 +25,25 @@ async def healthcheck():
 async def read_main():
     logger.info("In root path")
     return {"msg": "Hello World"}
+
+
+@index_router.get("/get_my_pixel_watches", response_model=List[schemas.PixelWatch])
+def get_my_pixel_watches(user: UserResponse = Depends(get_current_user), db: Session = Depends(get_db)):
+    user_pixel_watches = db.query(PixelWatch).join(user).filter(PixelWatch.user_id == user.user.id).all()
+
+    return user_pixel_watches
+
+
+@index_router.post("/watch_my_pixel")
+async def watch_my_pixel(params: WatchPixelParams,
+                         user: UserResponse = Depends(get_current_user),
+                         db: Session = Depends(get_db)):
+    db.add(PixelWatch(latitude=params.coordinates.latitude,
+                      longitude=params.coordinates.longitude,
+                      path=params.wrs_coordinates.wrs_path,
+                      row=params.wrs_coordinates.wrs_row,
+                      datetime=params.date,
+                      user_id=user.user.id,
+
+                      ))
```

Comment:
Fix ident + you can reformat it using jetbrains editor (reformat code) it works quite nice